pkg/crc/machine/start.go

```diff
@@ -385,6 +385,13 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		return nil, errors.Wrap(err, "Error waiting for apiserver")
 	}
 
+	// Remove this check when we have official 4.8 bundle
+	if strings.HasPrefix(crcBundleMetadata.GetOpenshiftVersion(), "4.8.") {
+		if err := cluster.EnsureSSHKeyPresentInTheCluster(ocConfig, constants.GetPublicKeyPath()); err != nil {
```

Comment:
I guess the worst case is:

* VM starts,
* crc updates the ssh key manually to the new one
* kubelet starts and starts the MCO
* MCO updates the ssh key to the old one
* crc updates the ssh key config map
* MCO updates the ssh key to the new one

Could this happen? MCO is slower to start than the apiserver for sure.

Reply:
@guillaumerose That's the reason we do it as soon as apiserver is responding, in this case MCO will take the latest config when it start to render it. What you described only possible if MCO started with old config. I am hoping this shouldn't happen at all

Reply:
> Should the MCO be scaled down to 0 by snc, then crc updates all the resources and scale up to 1 ?
I think also at the pull secret. Could it trigger a restart of kubelet if MCO goes too fast ?

This feels like more safer to do..

Reply:
For ssh key, it is fine I guess but for the pull secret it is more an issue. 
If the MCO triggers a restart of the kubelet because it detects a change of pull secret, it will add extra start time.

Reply:
Let's try this. We will see how it goes 👍